services/blueprint.yml.erb

```diff
@@ -1,7 +1,10 @@
 <%
   redirect_uri = "https://services-fabrik-broker.bosh-lite.com/manage/auth/cloudfoundry/callback"
-  def base64_template(prefix)
-    filename = "../services/#{prefix}-manifest.yml.ejs"
+  def base64_template(prefix, type)
+    if (type == "dashboard")
```

Comment:
Did you try directly using the "type" variable while constructing the filename ?

Or can you use case statements instead of else so that its more explicit why else is manifest?

Reply:
Agree. It'd be a good idea to change name to `../services/#{prefix}-#{type}.yml.ejs`